### PR TITLE
start en/messages.json source for add-on

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: en/messages.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
+{
+    "extensionDescription": {
+        "message": "Firefox Relay makes it easy to create email aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.",
+        "description": "Description of the extension. DO NOT TRANSLATE \"Firefox Relay\"."
```
Comment (peiying2) [inline en/messages.json:4]: Do you want to add `Relay` to the list of DO NOT Translate?

File: en/messages.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        "message": "Manage All Aliases"
+    },
+    "popupOnboardingPanel1Body": {
+        "message": "When the Firefox Relay icon appears on a website, select it to generate a new alias."
+    },
+    "popupOnboardingPanel2Headline": {
+        "message": "Don't see the icon?"
+    },
+    "popupOnboardingPanel2Body": {
+        "message": "Right-click (Windows) or Control-click (macOS) on form fields to access the context menu and generate an alias from there."
```
Comment (peiying2) [inline en/messages.json:88]: If we want to re-write this message to indicate we support all the platforms, let's do.

**Review by peiying2 — COMMENTED** [review]
Other than the string specific comments, let's be consistent with the curly quote vs. straight quote. Let's use curly quote throughout. This will also match the practice with the Fluent file.

Another is the branding, we have at least three ways of referring relay. Do we want to call it Firefox Relay Premium when it's mentioned in full?

**Review by groovecoder — COMMENTED** [review]
Updated to curly quote and re-wrote `popupOnboardingPanel2Body` to indicate support on all platforms.

What's the best way to designate "DO NOT TRANSLATE Relay" for the whole file?

**Comment by groovecoder** [comment]
On branding, I think "Firefox Relay" is the brand - not necessarily "Premium".

**Comment by peiying2** [comment]
> Updated to curly quote and re-wrote `popupOnboardingPanel2Body` to indicate support on all platforms.
> 
> What's the best way to designate "DO NOT TRANSLATE Relay" for the whole file?

I will get back to you to see if there is a way to do it. And if yes, we can put Firefox in the list as such.

**Comment by groovecoder** [comment]
Updated with more descriptions and notes to NOT translate product names.

**Review by peiying2 — APPROVED** [review]
Looks good. Thanks for providing the detailed context for the strings.

**Pull request merged**